packages/engine/lib/orchestrator/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "orchestrator"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+error = { path = "../../lib/error", features = ["spantrace"] }
+hash_engine = { path = "../..", default-features = false }
+#server = { path = "../server", artifact = "bin" }
```

Comment:
How come this is commented out?

Reply:
This was copied over from `cli`, I commented it out because this is for the upcoming artifact feature of cargo.

Reply:
```suggestion
```